text/trace/0000-sampling-propagation.md

```diff
@@ -0,0 +1,126 @@
+# Propagate head trace sampling probability
+
+Propose extending the W3C trace context `traceparent` to convey head trace sampling probability.
+
+## Motivation
+
+The head trace probability is useful in child contexts to be able to
+record the effective sampling probability in child spans.  This is
+documented in [OTEP 148](TODO), which establishes semantic conventions
```

Comment:
Missing link ;)

Reply:
I do not like to place links to active PRs, so I would wait for OTEP 148 to merge before finishing these. The link is to Dapper's inflationary sampler which requires knowing the probability of negative decisions in order to properly inflate the probability at a new root.